testing/tests/DevExpress.ui.widgets.pivotGrid/sortable.tests.js

```diff
@@ -983,18 +1000,20 @@ QUnit.module('\'useIndicator\' option', () => {
 
     QUnit.test('Indicator should not be shown on dragging to the same item at another sortable', function(assert) {
         $('#sortable').css('display', 'none');
-        $('<div id=\'sortable1\'><div id=\'second-group\' group=\'groupFilter\' class=\'group horizontal\' style=\'height: 150px\'><div class=\'test-container\'><div class=\'test-item\'>1</div><div class=\'test-item\'>2</div><div class=\'test-item\'>3</div></div></div>')
+        $('<div id=\'sortable1\'><div id=\'second-group\' group=\'groupFilter\' class=\'group horizontal\'><div class=\'test-container\'><div class=\'test-item\'>1</div><div class=\'test-item\'>2</div><div class=\'test-item\'>3</div></div></div>')
             .insertAfter('#sortable')
             .css({
                 width: '3000px',
                 height: '200px'
             });
-        $('<div id=\'sortable2\'><div id=\'second-group\' group=\'groupFilter\' class=\'group horizontal\' style=\'height: 150px\'><div class=\'test-container\'><div class=\'test-item\'>1</div><div class=\'test-item\'>2</div><div class=\'test-item\'>3</div></div></div>')
+        $('#sortable1 #second-group').css('height', '150px');
+        $('<div id=\'sortable2\'><div id=\'second-group\' group=\'groupFilter\' class=\'group horizontal\'><div class=\'test-container\'><div class=\'test-item\'>1</div><div class=\'test-item\'>2</div><div class=\'test-item\'>3</div></div></div>')
```

Comment:
Let's add more obvious formatting and use ``` ` ``` instead of the ``` ' ``` here:

```suggestion
        $(`
        <div id='sortable2'>
          <div id='second-group' group='groupFilter' class='group horizontal'>
            <div class='test-container'>
              <div class='test-item'>1</div>
              <div class='test-item'>2</div>
              <div class='test-item'>3</div>
            </div>
          </div>
        `);
```